### PR TITLE
Create `libexec/version.sh` for tarball dry run

### PR DESCRIPTION
**Pull request by tieujason330**
### What changes are proposed in this pull request?
- generate `libexec/version.sh` for using `--dryRun` 

### Why are the changes needed?

For using dry run flag for dev purposes

### CODE REVIEW

File: dev/scripts/src/alluxio.org/build/cmd/build.go [hunk 1]
```diff
@@ -109,6 +109,10 @@ func TarballF(args []string) error {
 		moduleMvnCmd := baseMvnCmd + " " + m.MavenArgs
 		if opts.dryRun {
 			log.Printf("Skip running maven command:\n%v", moduleMvnCmd)
+			// need to create libexec/version.sh
```
Comment (Xenorith) [inline dev/scripts/src/alluxio.org/build/cmd/build.go:112]: run script if version.sh file doesn't exist

**Comment by ssz1997** [comment]
I'm going to revert the docker change because I think I forgot to push my latest changes to that PR so that the current dockerifle doesn't work.

**Review by Xenorith — APPROVED** [review]
approved after reverting the docker related changes in profiles.yml due to revert that just merged

**Comment by tieujason330** [comment]
alluxio-bot, merge this please

**Comment by tieujason330** [comment]
alluxio-bot, merge this please

**Pull request merged**